src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -61,16 +62,12 @@ case class WorkflowDescriptor(id: WorkflowId,
   lazy val fileHasher: FileHasher = { wdlFile: WdlFile => SymbolHash(ioManager.hash(wdlFile.value)) }
 
   // GCS FS with the workflow working directory as root
-  val gcsFilesystem = for {
-    interface <- gcsInterface
-    fs <- Try(GcsFileSystem.instance(interface, wfContext.root))
-  } yield fs
+  lazy val gcsFilesystem = Try(GcsFileSystem.instance(gcsInterface, wfContext.root))
```

Comment:
I don't understand - why does `WorkflowDescriptor` have a `GcsInterface`? Is `WorkflowDescriptor` never used on `LocalBackend`?